Lib/linecache.py

```diff
@@ -1,10 +1,10 @@
+import importlib._bootstrap_external
```

Comment:
I don't really like importing this one at the very top and even before the docstring.

Ideally, this should be imported locally so to avoid possible circular references.